shared/js/loadApp.js

```diff
@@ -50,7 +68,8 @@ module.exports = function (callback) {
         .then(loadSource('ace/mode-javascript'))
         .then(loadSource('ace/ext-language_tools'))
         .then(loadSource('droplet/droplet-full'))
-        .then(loadSource('tooltipster/jquery.tooltipster'));
+        .then(loadSource('tooltipster/jquery.tooltipster'))
+        .then(loadExternalSource('https://www.google.com/jsapi'));
```

Comment:
It looks like this loads 5.8 KB of JavaScript, which stacks up favorably against dependencies like ace (330 KB minified) or just the qTip jQuery extension (43.5 KB minified).  It's close to md5.js at 6.5 KB.

I can't find evidence that it does anything else until you invoke `google.load` to pull in packages you need, which we do right before rendering the chart.